src/ansys/speos/core/kernel/client.py

```diff
@@ -129,26 +129,44 @@ class SpeosClient:
         By default, ``INFO``.
     logging_file : Optional[str, Path]
         The file to output the log, if requested. By default, ``None``.
+    speos_install_loc : Optional[str, Path]
+        location of Speos rpc executable
     """
 
     def __init__(
         self,
         host: Optional[str] = DEFAULT_HOST,
         port: Union[str, int] = DEFAULT_PORT,
+        version: str = LATEST_VERSION,
         channel: Optional[grpc.Channel] = None,
         remote_instance: Optional["Instance"] = None,
         timeout: Optional[int] = 60,
         logging_level: Optional[int] = logging.INFO,
         logging_file: Optional[Union[Path, str]] = None,
+        speos_install_loc: Optional[Union[Path, str]] = None,
```

Comment:
in the retrieve_speos_install_dir method, speos_rpc_loc is required to be str/Path, why uses None here.

Reply:
to work when using it on remote server
then we don't know the location